viewpost.php

```diff
@@ -0,0 +1,58 @@
+<?php
+include ('config/init.php');
+$post =getPostInformation($_REQUEST['postID']);
+siteHeader('blog * '.$post["title"].'','blog + inspiration');
+
+
+if ($post['nextPost'] == NULL) {
+    print "
+    <div class='postSubHeader'>
+    <h2>".$post["header"].":</h2>
+    </div>
+    <div class='postBody'>
+    ".$post["body"]."
+    <br><br>
+    <div class='blogCommentForm'>
+    <h3>Leave a comment:</h3>
+    <form action='comment.php?ID=$_REQUEST[postID]' method='post'>
+    Name: <input type='text' name='name'><br>
+    Email: <input type='test' name='email'> <br>
+    Comment: <br> <textarea name='body' placeholder='Add a comment...' rows='10' cols='50'></textarea><br>
+    <input type='submit' value='Submit'>
+    </form>
+    </div>
+    </div>
+    <div class='backNextOnViewPostPage'>
+    <table>
+    <tr>
+    <th><a href='Myblog.php'>back to blog + inspiration home</a></th>
+    <th><a href='comingSoon.php'>next post</a></th>
+    </tr>
+    </div>";
+}
+else {
```

Comment:
It looks like you're repeating a lot of code inside this if/else statement. It seems like you could probably pull the common code out so you only have to type it once, and then just put the dynamic part in the if statement. Aside from keeping your code cleaner, that makes it more maintainable. Imagine you have to come in and change something in the future. You might forget to change it in both places.